### ISSUE TITLE
Setup ARIA Page

### ISSUE THREAD

**Issue by michaeldev6** [michaeldev6/dev-a11y#5]
There should be a clear distinction between ARIA roles and ARIA attributes on the page. 

Similar to the WCAG page, where there is a filter, showing all, roles, groups, tags, and maybe even connect it with which ones are useful for each WCAG Item.

Still figuring out the enums and interfaces for the ARIA items.

Each one should also be a link to the more details page. Still thinking on the route structure.
